packages/loader/lib/index.js

```diff
@@ -56,57 +76,31 @@ mainContext = this; // eslint-disable-line no-undef
     return exports;
   }
 
-  let isNode =
-    typeof window === 'undefined' &&
-    typeof process !== 'undefined' &&
-    {}.toString.call(process) === '[object process]';
-
-  if (!isNode) {
-    Ember = this.Ember = this.Ember || {};
-  }
-
-  if (typeof Ember === 'undefined') {
-    Ember = {};
-  }
-
-  if (typeof Ember.__loader === 'undefined') {
-    registry = Object.create(null);
-    seen = Object.create(null);
-
-    define = function (name, deps, callback) {
-      let value = {};
+  require = function (name) {
+    return internalRequire(name, null);
+  };
 
-      if (!callback) {
-        value.deps = [];
-        value.callback = deps;
-      } else {
-        value.deps = deps;
-        value.callback = callback;
-      }
+  // eslint-disable-next-line no-unused-vars
+  define = function (name, deps, callback) {
+    var value = {};
 
-      registry[name] = value;
-    };
-
-    require = function (name) {
-      return internalRequire(name, null);
-    };
+    if (!callback) {
+      value.deps = [];
+      value.callback = deps;
+    } else {
+      value.deps = deps;
+      value.callback = callback;
+    }
```

Comment:
Do we actually need a branch here? Doesn't our babel setup always emit 3 args?

Reply:
Seems like we don't removing